Improved documentation about time stamped data (#719)
nexusformat/definitions: manual/source/strategies.rst

Strategies: Time-stamped data
#############################
*How should I store time-stamped data?*

Time-stamped data can be stored in both :ref:`NXlog` and :ref:`NXevent_data` structures. 
Of the two, :ref:`NXlog` is the most important one, :ref:`NXevent_data` is used only for storing neutron event data 
and :ref:`NXlog` would be used for storing any other time-stamped data, e.g. sample temperature, chopper top-dead-centre, 
motor position, detector data etc.

> Regarding the NeXus file structure to use, there is one simple rule: just use the standard NeXus file structure but replace 

Reviewer: ```suggestion
Regarding the NeXus file structure to use, there is one simple rule: just use the standard NeXus file structure but insert 
```

want.
Obviously in this simple example this does not gain us a lot, but it is easy to see that in a 
large dataset having appropriately placed cues can save significant computational time when looking 
up values in a certain time-stamp range. NeXus has actually borrowed the cueing table concept 

> from video file formats where it allows viewing software to quickly access your most favourite scene. 

Reviewer: ```suggestion
from video file formats where it allows viewing software to quickly access your favourite scene. 
```

> Correspondingly, cueing in NeXus allows you to quickly access your most favourate morsel of time stamped 

Reviewer: ```suggestion
Correspondingly, cueing in NeXus allows you to quickly access your favourite morsel of time stamped 
```
